WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -11,22 +14,55 @@
 import org.wordpress.android.widgets.OpenSansEditText;
 import org.wordpress.android.widgets.WPTextView;
 
-public class DialogUtils {
-    public static void showMyProfileDialog(Context context,
-                                           String title,
-                                           String initialText,
-                                           String hint,
-                                           boolean isMultiline,
-                                           final Callback callback) {
-        LayoutInflater layoutInflater = LayoutInflater.from(context);
+public class DialogUtils extends DialogFragment {
+
+    private static final String TITLE_TAG = "TITLE";
+    private static final String INITIAL_TEXT_TAG = "INITIAL-TEXT";
+    private static final String HINT_TAG = "HINT";
+    private static final String IS_MULTILINE_TAG = "IS-MULTILINE";
+
+    private static Context mContext;
+    private static Callback mCallback;
```

Comment:
Hi, I tried this and it worked :) I have one doubt, if we cast getActivity() to MyProfileActivity then it couldn't not be used by other activities in future. Will that be fine?
And sorry for late reply